app/src/Cells.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+function Cells(props) {
```

Comment:
A couple of small things here:

1. I'd name this component `Cell` instead of `Cells` since it represents on square on the board. 
2. If you want to, you can use object destructuring to avoid needing to use `props.` multiple times.

```js
function Cell({click, value})
```